src/main/java/com/smartbear/plugins/swaggerhub/PublishToHubAction.java

```diff
@@ -90,18 +88,8 @@ private boolean publishApi(RestService restService) throws IOException {
                 }
             }
 
-            Class exporterClass = Class.forName("com.smartbear.swagger.Swagger2Exporter");
-            Object exporter = exporterClass.getConstructor(WsdlProject.class).newInstance(restService.getProject());
-
-            Method method;
-            try {
-                method = exporterClass.getMethod("exportToFolder", String.class, String.class, String.class, RestService[].class, String.class);
-            } catch (NoSuchMethodException e) {
-                method = exporterClass.getMethod("exportToFolder", String.class, String.class, String.class, AbstractRestService[].class, String.class);
-            }
-
-            String result = (String) method.invoke(exporter, Files.createTempDir().getAbsolutePath(),
-                    versionId, "json", new RestService[]{restService}, restService.getBasePath());
+            Swagger2Exporter exporter = new Swagger2Exporter(restService.getProject());
```

Comment:
Why is this Swagger2Exporter only and not OpenAPI3Exporter?

Reply:
We have separate task RIA-7261